src/test/java/org/folio/rest/support/http/ResourceClient.java

```diff
@@ -160,22 +170,26 @@ public Response getById(UUID id)
     return getCompleted.get(5, TimeUnit.SECONDS);
   }
 
-  public void delete(UUID id)
-    throws MalformedURLException,
-    InterruptedException,
-    ExecutionException,
-    TimeoutException {
+  public Response deleteIfPresent(String id) throws MalformedURLException,
+    InterruptedException, ExecutionException, TimeoutException {
 
     CompletableFuture<Response> deleteFinished = new CompletableFuture<>();
 
     client.delete(urlMaker.combine(String.format("/%s", id)),
       StorageTestSuite.TENANT_ID, ResponseHandler.any(deleteFinished));
 
-    Response response = deleteFinished.get(5, TimeUnit.SECONDS);
+    return deleteFinished.get(5, TimeUnit.SECONDS);
+  }
+
+  public void delete(UUID id) throws MalformedURLException, InterruptedException,
+    ExecutionException, TimeoutException {
+
+    Response response = deleteIfPresent(id != null ? id.toString() : null);
 
     assertThat(String.format(
       "Failed to delete %s %s: %s", resourceName, id, response.getBody()),
-      response.getStatusCode(), is(204));
+      response.getStatusCode(),
+      is(204));
```

Comment:
Why has this line been split?

Reply:
For better readability. Now each param is on a separate line, i guess it is more readable now.

Reply:
Why does having the property and the matcher on separate lines make this more readable?

Reply:
at least you can easily see which parameters are passed to the method. Previously it was like the getStatusCode() is a parameter of String.format(...).

Reply:
I really don't follow. 

The only changes reported here show splitting the `response.getStatusCode()` and `is(204)` onto separate lines, and I don't understand why that is more readable. 

What am I missing?

Reply:
I've reverted this change.